.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -29,8 +29,11 @@ React Native version:
 Describe what you expected to happen:
 
 
-Snack, code example, or link to a repository:
+Snack, screenshot, code example, or link to a repository:
```

Comment:
Place code example first, to encourage actual code to be included before screenshots.

```suggestion
Snack, code example, screenshot, or link to a repository:
```